common/raw_hashtable.h

```diff
@@ -1449,7 +1454,7 @@ auto BaseImpl<InputKeyT, InputValueT, InputKeyContextT>::GrowToNextAllocSize(
       ssize_t old_hash_index = hash.ExtractIndexAndTag<7>().first &
                                ComputeProbeMaskFromSize(old_size);
       if (LLVM_UNLIKELY(old_hash_index != group_index)) {
-        probed_indices.push_back(old_index);
+        probed_indices.emplace_back(old_index, hash);
```

Comment:
I would suggest:
```suggestion
        probed_indices.push_back({old_index, hash});
```

We consistently see `push_back` out-performing `emplace_back` and so prefer it strongly.

Reply:
Using `push_back` now.
One day I would be interested to know why that may be the case...

Reply:
I wrote up some very rough attempts to write up what I keep seeing here starting with this post: https://hachyderm.io/@chandlerc/113649754221959997

Reply:
(just to be clear, the start of that thread was a completely separate use of `emplace_back` that very surprisingly had a performance cost)